### ISSUE TITLE
How to change the shipping address using the API ?

### ISSUE THREAD

**Issue by pesquibr** [printdotio/printio-ios-sdk#361]
Hi all. I'm working in an app that has authentication. Every authenticated user in the app can buy products on the Shop. I'm getting a problem with the address entered by the user. If an user sets the shipping address, the same address is available for any other user in my app. But this is not the behavior I was expecting, I would that information to be private for each user. Is there a way in the API to load, save or clean the address information ? Thanks.

**Comment by perisicboro** [comment]
@pesquibr What do you mean by multiple users will use app? App is installed on one iPhone, so you say that more than one user will use it?

**Comment by pesquibr** [comment]
My app is similar to the Facebook App, that has a login screen. So you can authenticate with different users in the same device.

**Comment by perisicboro** [comment]
@pesquibr Ok, we don't have any method that can handle this right now. It's not only about address, we should also consider what to do with items in shopping cart for different users. I can't give you exactly the date when we will work on this.

**Comment by perisicboro** [comment]
@pesquibr For now I can make methods, one to clear shopping cart, and other to remove all addresses for database, if that means something for you?

**Comment by pesquibr** [comment]
The drawback is that we'll lose the user's information, but I think that is better than show a wrong information.

**Comment by perisicboro** [comment]
@pesquibr I added three new methods, which you can use in right moment:

```Objective-C
[PrintIO clearShoppingCart];
[PrintIO clearBillingAddresses];
[PrintIO clearShippingAddresses];
```
It will be available in ```1.0.45-dev``` build, later today.

**Comment by pesquibr** [comment]
@perisicboro Great. Thanks a lot.
